src/lib/components/layouts/DocumentBrowser.svelte

```diff
@@ -154,7 +154,14 @@
             <PageToolbar>
               <Flex slot="right">
                 <div style:flex="1 1 auto">
-                  <Search name="q" {query} placeholder={uiText.search} />
+                  <Search name="q" {query} placeholder={$_(uiText.search)}>
+                    <span slot="help">
```

Comment:
Just curious, why a slot instead of just rendering beneath?

Reply:
Mostly for flexibility. This help applies to searching documents, but we use this component to search projects, add-ons and document text, and we might want different help text in each place.